jsapp/js/lists/forms.es6

```diff
@@ -19,17 +23,42 @@ class FormsSearchableList extends React.Component {
           assetType: COMMON_QUERIES.s,
         },
         filterTags: COMMON_QUERIES.s,
-      })
+      }),
+      showModal: false,
+      limits: 0,
     };
+    this.data = this.data.bind(this);
   }
-  componentDidMount () {
+
+  componentDidMount() {
     this.searchSemaphore();
   }
-  render () {
+
+  data = (limit) => {
+    this.setState({limits: limit});
+    const allCookies = cookies.getAll();
+    const isLimitCookie = Object.keys(allCookies).find(
+      (key) => key === 'overLimitsCookie'
+    );
+    if (isLimitCookie === undefined && limit > 0) {
+      this.setState({showModal: true});
+      var dateNow = new Date();
+      var expireDate = new Date(dateNow.setDate(dateNow.getDate() + 1));
+      cookies.set('overLimitsCookie', {
+        expires: expireDate,
+      });
+    }
+  };
+
+  render() {
     return (
       <div className={styles.myProjectsWrapper}>
         <div className={styles.myProjectsHeader}>
-          <ViewSwitcher selectedViewUid={HOME_VIEW.uid}/>
+          <LimitModal
+            show={this.state.showModal}
```

Comment:
The last thing is that this modal shouldn't show if Stripe is disabled. It should also prevent the `getSubscription` call and any other fetches from Stripe endpoints without Stripe. You can check that using `envStore.data.stripe_public_key`; if it's defined and not null, Stripe is enabled.